Reorganize project ReadMe + Additional supporting documents (#11725)
endless-sky/endless-sky: README.md

- Make sure to follow our [community guide](docs/COMMUNITY.md)

## Contributing Guidelines
We welcome contributions in code, content, testing, and documentation.
- Please begin with our [contributing guidelines](docs/CONTRIBUTING.md)
- Review [governance](docs/GOVERNANCE.md) before making contributions.
- New contributors may also consult the [FAQ](docs/FAQ.md) for common questions.

As a free and open source game, Endless Sky is the product of many people's work. Contributions of artwork, storylines, and other writing are most in-demand, though there is a loosely defined [roadmap](https://github.com/endless-sky/endless-sky/wiki/DevelopmentRoadmap). Those who wish to [contribute](docs/CONTRIBUTING.md) are encouraged to review the [wiki](https://github.com/endless-sky/endless-sky/wiki), and to post in the [community-run Discord](https://discord.gg/ZeuASSx) beforehand. Those who prefer to use Steam can use its [discussion rooms](https://steamcommunity.com/app/404410/discussions/) as well, or GitHub's [discussion zone](https://github.com/endless-sky/endless-sky/discussions).

## Governance

> Governance of Endless Sky falls to a system of Admins, Maintainers, Developers, and Reviewers. While being moderated by our "Oathkeeper" team. Details of members as well as descriptions of roles are available in [governance](docs/GOVERNANCE.md)

Author: This feels a bit clunky to me, how about:

```suggestion
Governance of Endless Sky is managed by a system of Admins, Maintainers, Developers, and Reviewers, with the whole moderated by our Oathkeeper team. Details of members as well as descriptions of roles are available in [governance](docs/GOVERNANCE.md)
```

> Governance of Endless Sky falls to a system of Admins, Maintainers, Developers, and Reviewers. While being moderated by our "Oathkeeper" team. Details of members as well as descriptions of roles are available in [governance](docs/GOVERNANCE.md)

Reviewer: I did like "falls to", so how about this:
```suggestion
Governance of Endless Sky falls to a system of Admins, Maintainers, Developers, and Reviewers, with the whole being moderated by our Oathkeeper team. Details of members as well as descriptions of roles are available in [governance](docs/GOVERNANCE.md)
```
If you use "is managed by", it would work better like
```suggestion
Endless Sky is managed by a system of Admins, Maintainers, Developers, and Reviewers, with the whole moderated by our Oathkeeper team. Details of members as well as descriptions of roles are available in [governance](docs/GOVERNANCE.md)
```

If you’re instead looking to learn about the game itself (installation, gameplay basics, or system requirements), please see the [game readme](GAME.md)

## Table of Contents
- [Community and Membership](#community-and-membership)
- [Contributing Guidelines](#contributing-guidelines)
- [Governance](#governance)
- [Building from Source](#building-from-source)
- [Licensing](#licensing)
- [FAQ](#frequently-asked-questions)
- [Documentation](#all-documentation)

## Community and Membership
Endless Sky is supported by a diverse group of maintainers, contributors, and community members.
- See [members](docs/MEMBERS.md) for a list of organization members
- New contributors are welcome and may advance to broader roles over time through demonstrated participation and collaboration.

> - Community discussions take place primarily on GitHub, with supplementary conversations occurring through [Discord](https://discord.gg/ZeuASSx).

Author: How about just rewriting the line to be more neutral and also list a few other places?
```suggestion
- Community discussions take place on [GitHub](https://github.com/endless-sky/endless-sky/discussions), [Discord](https://discord.gg/ZeuASSx), [Steam](https://steamcommunity.com/app/404410/discussions/) and [Reddit](https://www.reddit.com/r/endlesssky/).
```
